### PR TITLE
Fixed glass panes and iron bars, not connecting to cloth and stonebrick 

### PR DESCRIPTION
**Pull request by duckdoom5**
Also edited sprinting method and fixed mouse click not being fired

Signed-off-by: duckdoom5 j_e_l_l_e@hotmail.com

### CODE REVIEW

File: src/main/java/net/minecraft/src/EntityClientPlayerMP.java [hunk 1]
```diff
@@ -62,6 +62,9 @@ public void onUpdate() {
 				setSprinting(true);
 				this.sendQueue.addToSendQueue(new Packet19EntityAction(this, 4));
 			}
+		}else if(this.isSprinting() && !canSprint){
+		    setSprinting(false);
+		    runToggle = false;
```
Comment (ZNickq) [inline src/main/java/net/minecraft/src/EntityClientPlayerMP.java]: @Afforess can you confirm this is right? It looks fine to me, but you implemented the run stuff :P

Comment (duckdoom5) [inline src/main/java/net/minecraft/src/EntityClientPlayerMP.java]: oh crap, canSprint misses () it has to be canSprint(), wait a sec, I will change it

**Comment by duckdoom5** [comment]
If you guys don't want to merge all of it then please fix the glass pane/ iron bar thing, it really looks stupid and it normally works in minecraft

**Comment by lukespragg** [comment]
I'm sure someone will get to it duckdoom5, patience. ;)

**Comment by duckdoom5** [comment]
I just said it because I noticed all other requests where pulled :P

**Comment by ZNickq** [comment]
Thanks, sorry about the delay, i wasn't sure about adding a new file for such a small bug fix, but it's not a huge issue ;)

**Pull request merged**